### ISSUE TITLE
[Bug]: technical variables summary never shows

### ISSUE THREAD

**Issue by sof202** [ejh243/BrainFANS#188]
### What happened?

In the QC.rmd, Section 1.1 will never show summaries of samples categorised by [insert `techVar` item here]

### How can the bug be reproduced?

Run the QC pipeline and look at QC.html. Section 1.1 will always be empty.

**Comment by sof202** [comment]
## Why?

The code associated with `techVar` is different from `bioVar`:

`techVar`:

```R
for(item in techVar){
  tabTemp<-as.data.frame(table(QCmetrics[,item]))
  if(nrow(tabTemp) < 2){
    pander(
      t(tabTemp),
      caption = paste("Summary of samples categorised by", item),
      row.names = c("Variable", "Number of Samples")
    )
  }
}
```

`bioVar`:

```R
for(item in bioVar){
  pander(
    as.data.frame(table(QCmetrics[, item])),
    caption = paste("Summary of samples categorised by", item),
    col.names = c("Variable", "Number of Samples")
  )
}
```

-----------------

## Problem

Notice how `techVar`'s section of code is almost identical, but it requires `tabTemp` to have less than 2 rows. The only way for this code to render anything on QC.html therefore is for `tabTemp` to have 1 row. However, the only way for `tabTemp` to have a single row is if every value in the column extracted from `QCmetrics` is the same. This is where the problem lies, variables are removed from analysis if all values are the same now thanks to PR #178. As such, `techVar` does not include any columns that can spit out a `tabTemp` that satisfies this `if` statement (and even if it did, such a table would be rather useless).

I presume that this check was implemented to **not** show variables in `techVar` that had all values the same originally (though this would require the `<` to be converted into `>`). But this still doesn't explain why `bioVar` is not handled in this way.

------------------

## Solutions

We can either:

- Remove the conditional check in the `techVar` block
- Convert the `<` to a `>` in the conditional check for the `techVar` block
- Convert the `<` to a `>` in the conditional check for the `techVar` block and add it to the `bioVar` block

**Comment by sof202** [comment]
This will be fixed by the PR addressing #216. Closing the issue.
